vulnerabilities/models.py

```diff
@@ -261,53 +261,6 @@ def update_or_create(self):
             )
 
 
-class Importer(models.Model):
```

Comment:
Why not adding an ``ImporterRun`` model right away to log each run there?

Reply:
Logging will be a different module as we discussed earlier. Other than printing to stderr, there's no logging here (so that nothing interferes with our logging module)

Reply:
https://github.com/nexB/vulnerablecode/issues/590